python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -0,0 +1,252 @@
+
+    @anchor LoadReferenceObjectsTask_
+
+    @section meas_algorithms_loadReferenceObjects_Contents Contents
+
+     - @ref meas_algorithms_loadReferenceObjects_Purpose
+     - @ref meas_algorithms_loadReferenceObjects_Initialize
+     - @ref meas_algorithms_loadReferenceObjects_IO
+     - @ref meas_algorithms_loadReferenceObjects_Schema
+     - @ref meas_algorithms_loadReferenceObjects_Config
+
+    @section meas_algorithms_loadReferenceObjects_Purpose  Description
+
+    Abstract base class for tasks that load objects from a reference catalog
+    in a particular region of the sky.
+
+    Implementations must subclass this class, override the loadObjectsInBBox method,
+    and will typically override the value of ConfigClass with a task-specific config class.
+
+    @section meas_algorithms_loadReferenceObjects_Initialize   Task initialisation
+
+    @copydoc \_\_init\_\_
+
+    @section meas_algorithms_loadReferenceObjects_IO       Invoking the Task
+
+    @copydoc loadObjectsInBBox
+
+    @section meas_algorithms_loadReferenceObjects_Schema       Schema of the reference object catalog
+
+    Reference object catalogs are instances of lsst.afw.table.SimpleCatalog with the following schema
+    (other fields may also be present):
+    - coord: position of star on sky (an lsst.afw.coord.IcrsCoord)
+    - centroid: position of star on an exposure, if relevant (an lsst.afw.Point2D)
+    - hasCentroid: is centroid usable?
+    - *referenceFilterName*_flux: brightness in the specified reference catalog filter: 10^(-0.4*mag)
+    - *referenceFilterName*_fluxSigma (optional): brightness standard deviation;
+        omitted if no data is available; possibly nan if data is available for some objects but not others
+    - *cameraFilterName*_camera_flux: brightness in specified camera filter (magnitude)
+    - *cameraFilterName*_camera_fluxSigma (optional): brightness standard deviation
+        in specified camera filter (magnitude); omitted if no data is available;
+        possibly nan if data is available for some objects but not others
+    - default_flux (optional): brightness to use if no camera filter is available (magnitude);
+        omitted unless defaultFilter is specified in the config
+    - default_fluxSigma (optional): brightness standard deviation to use if no camera filter is available
+        (magnitude); omitted unless defaultFilter is specified in the config and the corresponding
+        fluxSigma field exists
+    - photometric (optional): is the object usable for photometric calibration?
+    - resolved (optional): is the object spatially resolved?
+    - variable (optional): does the object have variable brightness?
+
+    @section meas_algorithms_loadReferenceObjects_Config       Configuration parameters
+
+    See @ref LoadReferenceObjectsConfig for a base set of configuration parameters.
+    Most subclasses will add configuration variables.
+    """
+    __metaclass__ = abc.ABCMeta
```

Comment:
Are we getting anything out of the `abc` module that we couldn't just get from doing `raise NotImplementedError`() ourselves?  Or do you think this is just more idiomatic?